cluster/addons/fluentd-gcp/podsecuritypolicies/event-exporter-psp.yaml

```diff
@@ -18,7 +18,7 @@ spec:
   allowPrivilegeEscalation: false
   volumes:
   - 'hostPath'
-  - 'secret'
```

Comment:
keep secret in the list for all of these for now... otherwise existing deployments/replicasets/pods will be unhappy on upgrade